model/swagger.yml

```diff
@@ -30,6 +32,36 @@ servers:
     description: production server
 
 paths:
+  /healthcheck:
+    get:
+      tags:
+        - 5. healthcheck
+      summary: |
+        returns payload of system health information. Detection of any fatal issues results in a non-200 response. The primary intention of this endpoint is to provide an accurate assessment of the service to determine if ECS/Fargate should instantiate a new instance of the task.
+      operationId: healthcheck
+      responses:
+        '200':
+          description: The system checked out as healthy, returning a JSON payload of results
+          content:
+            "*":
+              schema:
+                type: object
+                additionalProperties: true
+        '400':
+          $ref: "#/components/responses/Error"
+        '404':
+          $ref: "#/components/responses/Error"
+        '418':
```

Comment:
Is HTTP418 really the way to go here (vs 200?).  The request was successful, so if we have any way to respond to the content of the response JSON (say, looking for the presence of `"healthy": true`), that would be preferable over abusing the status code imho.

Reply:
Roger that - have approved on the basis of the added comment